samples/transform_feedback_separated.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<!-- Ported from the OpenGL Samples Pack: https://github.com/g-truc/ogl-samples/blob/master/tests/gl-320-transform-feedback-separated.cpp -->
+<html lang="en">
+
+<head>
+    <title>WebGL 2 Samples - transform_feedback_separated</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div id="info">WebGL 2 Samples - transform_feedback_separated</div>
+
+    <script id="vs-transform" type="x-shader/x-vertex">
+        #version 300 es
+        #define POSITION_LOCATION 0
+
+        precision highp float;
+        precision highp int;
+
+        uniform mat4 MVP;
+        layout(location = POSITION_LOCATION) in vec4 position;
+
+        out vec4 v_color;
+
+        void main()
+        {
+            gl_Position = MVP * position;
+            v_color = vec4(clamp(vec2(position), 0.0, 1.0), 0.0, 1.0);
+        }
+    </script>
+
+    <!-- This fragment shader will not be called since it is used when the rasterizer is disabled. -->
+    <script id="fs-transform" type="x-shader/x-fragment">
+        #version 300 es
+        precision highp float;
+        precision highp int;
+
+        in vec4 v_color;
+        out vec4 color;
+
+        void main()
+        {
+            color = v_color;
+        }
+    </script>
+
+    <script id="vs-feedback" type="x-shader/x-vertex">
+        #version 300 es
+        #define POSITION_LOCATION 0
+        #define COLOR_POSITION 3
+
+        precision highp float;
+        precision highp int;
+
+        layout(location = POSITION_LOCATION) in vec4 position;
+        layout(location = COLOR_POSITION) in vec4 color;
+
+        out vec4 vColor;
+
+        void main()
+        {
+            gl_Position = position;
+            vColor = color;
+        }
+    </script>
+
+    <script id="fs-feedback" type="x-shader/x-fragment">
+        #version 300 es
+        precision highp float;
+        precision highp int;
+
+        in vec4 vColor;
+
+        out vec4 color;
+
+        void main()
+        {
+            color = vColor;
+        }
+    </script>
+
+    <script src="utility.js"></script>
+    <script>
+    (function () {
+        'use strict';
+
+        // -- Init Canvas
+        var canvas = document.createElement('canvas');
+        canvas.width = Math.min(window.innerWidth, window.innerHeight);
+        canvas.height = canvas.width;
+        document.body.appendChild(canvas);
+
+        // -- Init WebGL Context
+        var gl = canvas.getContext('webgl2', { antialias: false });
+        if (!gl) {
+            gl = canvas.getContext('experimental-webgl2', { antialias: false });
```

Comment:
Note: there's been agreement among the browser vendors that we're not going to expose an 'experimental-webgl2' context type. They'll all expose 'webgl2' behind a flag until it's ready to ship, at which point it'll be exposed unconditionally. So throughout these samples, any fetching of the 'experimental-webgl2' context type can and should be removed (since we shouldn't encourage web developers to follow this pattern).